app/economy/management/commands/fix_earnings.py

```diff
@@ -0,0 +1,157 @@
+            contribution = contributions.get(pk=earning.source_id)
+            if contribution.subscription.amount_per_period_minus_gas_price:
+                earning.token_value = contribution.subscription.amount_per_period_minus_gas_price
+
+                if contribution.success == False:
+                    earning.success = False
+                earning.save()
+        except:
+            print(f'unable to find contribution {earning.source_id}')
+
+
+def updateKudosEarnings (earnings):
+    print("===========================================")
+    print(f"Total Earning Kudos Count: {earnings.count()}")
+    kudos_transfers = KudosTransfer.objects.all()
+    for earning in earnings:
+        try:
+            kudo = kudos_transfers.get(pk=earning.source_id)
+            if kudo.amount:
+                earning.token_value = kudo.amount
+                earning.save()
+        except:
+            print(f'unable to find kudos {earning.source_id}')
+
+
+
+def updateBountyEarnings (earnings):
+    print("===========================================")
+    print(f"Total Earning Bounties Fulfillment Count: {earnings.count()}")
+    bounty_fulfillments = BountyFulfillment.objects.all()
+    for earning in earnings:
+        try:
+            bounty_fulfillment = bounty_fulfillments.get(pk=earning.source_id)
+            if bounty_fulfillment.payout_amount:
+                earning.token_value = bounty_fulfillment.payout_amount
+                earning.save()
+        except:
+            print(f'unable to find Bounty {earning.source_id}')
+
+
+def updateTipEarnings (earnings):
+    print("===========================================")
+    print(f"Total Earning Tips Count: {earnings.count()}")
+    tips = Tip.objects.all()
+    for earning in earnings:
+        try:
+            tip = tips.get(pk=earning.source_id)
+            if tip.amount:
+                earning.token_value = tip.amount
+                earning.save()
+        except:
+            print(f'unable to find Tip {earning.source_id}')
+
+
+def fixBountyFulfillmentAmount ():
+    print("===========================================")
+    bounty_fulfillments = BountyFulfillment.objects.filter(payout_amount__gt=1000000)
+
+    print(f"Fixing Amount for {bounty_fulfillments.count()}")
+
+    tokens = Token.objects.filter(approved=True)
+
+    for bounty_fulfillment in bounty_fulfillments:
+        token_name = bounty_fulfillment.token_name
+        token = tokens.filter(symbol=token_name).first()
+        if token:       
+            bounty_fulfillment.payout_amount = bounty_fulfillment.payout_amount / (10**token.decimals)
+            bounty_fulfillment.save()
+        else:
+            print(f'error: unable to find {token_name}')
+
+
+class Command(BaseCommand):
+    """Define the management command to fix up earnings."""
+
+    def handle(self, *args, **options):
+        """Fix earnings."""
+
+        print("Pulling in every Earning")
+        # e = Earning.objects.all()
```

Comment:
Did you mean to leave this commented code in?

Reply:
Ah that is dead code! not needed